src/DataTable/Actions.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { MoreVert } from '../../icons';
+import {
+  Button, Dropdown, useWindowSize, Icon, breakpoints,
+} from '..';
+
+export const DROPDOWN_BUTTON_TEXT = 'More actions';
+export const SMALL_SCREEN_DROPDOWN_BUTTON_TEXT = 'Actions';
+
+// eslint-disable-next-line react/prop-types
+const CustomToggle = React.forwardRef(({ children, onClick }, ref) => (
+  <Button
+    ref={ref}
+    variant="tertiary"
+    onClick={(e) => {
+      e.preventDefault();
+      onClick(e);
+    }}
+  >
+    {children}
+  </Button>
+));
+
+const Actions = ({
```

Comment:
I don't have a better suggestion, not noting: the name of this component made its purpose a little vague to me.

Reply:
Am I correct in thinking this component or some form of it is something generic outside datatable that we can add? Like `ActionRow` that can handle overflows, etc?

Reply:
Yeah, this form of it *is* pretty generic and we could make it more widely available. 
I don't love `ActionRow` because it doesn't take up a whole row? But I don't have a better idea.

Reply:
I'm gonna call it `CollapseableButtonGroup`